test_conformance/c11_atomics/host_atomics.h

```diff
@@ -175,7 +243,7 @@ CorrespondingType host_atomic_exchange(volatile AtomicType *a, CorrespondingType
     else
         return InterlockedExchange(reinterpret_cast<volatile LONG *>(a), c);
```

Comment:
Some of Windows CI/CD checks failed here, below correction works for me (build only)

```
    if constexpr (sizeof(CorrespondingType) == 2)
    {
        return InterlockedExchange16(
            reinterpret_cast<volatile SHORT*>(a),
            static_cast<SHORT>( static_cast<std::uint16_t>(c) )
        );
    }
    else
    {
        return InterlockedExchange(
            reinterpret_cast<volatile LONG*>(a),
            static_cast<LONG>( static_cast<std::uint32_t>(c) )
        );
    }

```